frontend/src/views/Collection/utils.tsx

```diff
@@ -114,13 +114,20 @@ export function getIsPublishable(datasets: Array<Dataset>): boolean {
   return (
     datasets?.length > 0 &&
     datasets.every((dataset) => {
-      const numOfAssets = dataset.dataset_assets.length;
+      // const numOfAssets = dataset.dataset_assets.length;
       const numOfDeployments = dataset.dataset_deployments.length;
 
-      return (
-        numOfDeployments === 1 &&
-        numOfAssets >= Object.keys(DATASET_ASSET_FORMAT).length
+      // return (
```

Comment:
nit: Maybe add TODO for other readers to know why it's commented out 😄 ? Thanks!